models/schema.sql

```diff
@@ -10,4 +10,60 @@ Create TABLE Users(
   updated_at DATETIME NOT NULL
 );
 
+--Scales
+DROP TABLE IF EXISTS Scales;
+CREATE TABLE Scales(
+  scale_id INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  created_at VARCHAR(255) NOT NULL,
+  user_id INT NOT NULL,
+  FOREIGN KEY (user_id) REFERENCES Users(user_id)
+    ON DELETE CASCADE
+);
+
+--Notes
+DROP TABLE IF EXISTS Notes;
+CREATE TABLE Notes(
+  note_id INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
+  note VARCHAR(255) NOT NULL,
+  priority INT NOT NULL,
+  octave INT NOT NULL
+);
+
+INSERT INTO Notes (note, priority, octave) values ("C", 1, 1);
+INSERT INTO Notes (note, priority, octave) values ("Db", 2, 1);
+INSERT INTO Notes (note, priority, octave) values ("D", 3, 1);
+INSERT INTO Notes (note, priority, octave) values ("Eb", 4, 1);
+INSERT INTO Notes (note, priority, octave) values ("E", 5, 1);
+INSERT INTO Notes (note, priority, octave) values ("F", 6, 1);
+INSERT INTO Notes (note, priority, octave) values ("Gb", 7, 1);
+INSERT INTO Notes (note, priority, octave) values ("G", 8, 1);
+INSERT INTO Notes (note, priority, octave) values ("Ab", 9, 1);
+INSERT INTO Notes (note, priority, octave) values ("A", 10, 1);
+INSERT INTO Notes (note, priority, octave) values ("Bb", 11, 1);
+INSERT INTO Notes (note, priority, octave) values ("B", 12, 1);
+
+INSERT INTO Notes (note, priority, octave) values ("C", 13, 2);
+INSERT INTO Notes (note, priority, octave) values ("Db", 14, 2);
+INSERT INTO Notes (note, priority, octave) values ("D", 15, 2);
+INSERT INTO Notes (note, priority, octave) values ("Eb", 16, 2);
+INSERT INTO Notes (note, priority, octave) values ("E", 17, 2);
+INSERT INTO Notes (note, priority, octave) values ("F", 18, 2);
+INSERT INTO Notes (note, priority, octave) values ("Gb", 19, 2);
+INSERT INTO Notes (note, priority, octave) values ("G", 20, 2);
+INSERT INTO Notes (note, priority, octave) values ("Ab", 21, 2);
+INSERT INTO Notes (note, priority, octave) values ("A", 22, 2);
+INSERT INTO Notes (note, priority, octave) values ("Bb", 23, 2);
+INSERT INTO Notes (note, priority, octave) values ("B", 24, 2);
```

Comment:
Currently this should probably go into the seeds.sql file. If you want to use one file for both schema and seeds I would rename this file to schemaAndSeeds.sql and delete the seeds.sql file.